### PR TITLE
Dynamic rendering of Recently Accessed files

### PR DESCRIPTION
**Pull request by Aniruddhmuley2001**
@meet59patel @HemangNakarani 
Have added the functionality as expected from issue #11 . The recently accessed documents are dynamically rendered, using React Functional Components.

Issue faced: In the present scenario, I could not generate the clickable link in the filename section. But will it work if another Column for the link is created?

### CODE REVIEW

File: src/components/dashboard/RecentlyAccessed.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    filename: 'filename2',
+    lastAccess: 'Mon, 13 Oct 2020, 12:45:10 IST'
+},{
+    filename: 'filename3',
+    lastAccess: 'Mon, 12 Oct 2020, 18:00:00 IST'
+}];
+
+const header = ["Filename", "Last Accessed"];
+
+const RecentlyAccesed = () => {
```
Comment (meet59patel) [inline src/components/dashboard/RecentlyAccessed.js:17]: Can you just fix this minor typo of `RecentlyAccessed` ?
The same change would go in the last line of same file - at `export default` part

**Review by meet59patel — CHANGES_REQUESTED** [review]
Great work, especially you got the point of using another object for temporarily showing the data 💯
This would be helpful when we fetch data from Backend as I've explained earlier.

Just fix the typo and I'll merge the PR 😄

**Comment by meet59patel** [comment]
@Aniruddhmuley2001 regarding your links doubt: Try doing something like this: 

- 
image
- 
image

I'll prefer if the links are there like this.

**Comment by Aniruddhmuley2001** [comment]
Have made the required changes.
Feature added: The file in the table will open on a different tab, without compromising with the ESLint property of jsx-no-target-blank property of the whole file, by disabling it only for one line in RecentlyAccessed.js, filling the file name in the table.

**Review by meet59patel — APPROVED** [review]
Great going! 🚀

**Comment by meet59patel** [comment]
give @Aniruddhmuley2001 25 points

**Pull request merged**